lib/bookingsync/api/client.rb

```diff
@@ -209,7 +214,7 @@ def paginate(path, options = {}, &block)
               first_request = false
             end
             break unless response.relations[:next]
-            response = response.relations[:next].get
+            response = response.relations[:next].call({}, method: request_method)
```

Comment:
makes sense and seems not to be braking methods signatures
might blow on nils if someone forced it somewhere, but looks good